functionary/core/api/v1/serializers/team.py

```diff
@@ -21,6 +22,7 @@ class Meta:
         model = Team
         fields = ["id", "name", "environments"]
 
+    @extend_schema_field(TeamEnvironmentSerializer(many=True))
```

Comment:
Unrelated to the rest of the PR.  This is just fixing an error I found in the swagger docs.